apps/prairielearn/src/ee/pages/lti13CourseNavigation/lti13CourseNavigation.html.ts

```diff
@@ -44,81 +44,74 @@ export function Lti13CourseNavigationInstructor({
+
+                    <label class="form-label" for="connect_course">
+                      Connect ${courseName} with PrairieLearn course:
                     </label>
                     <select
-                      class="custom-select"
-                      id="connect_course_instance"
-                      name="unsafe_course_instance_id"
+                      id="connect_course"
+                      class="form-select mb-3"
+                      name="unsafe_course_id"
+                      hx-get="${originalUrl}/course_instances"
+                      hx-include="#link_form"
+                      hx-target="#course_instances"
                     >
-                      <option value="" disabled selected>
-                        Select an existing course instance...
-                      </option>
-                      ${courses.map((course) => {
-                        const course_cis = course_instances.filter(
-                          (ci) => ci.course_id === course.id,
-                        );
-                        return html`
-                          <optgroup label="${course.short_name}: ${course.title}">
-                            ${course_cis.map((ci) => {
-                              return html`<option value="${ci.id}">
-                                ${course.short_name} -- ${ci.long_name} (${ci.short_name})
-                              </option>`;
-                            })}
-                          </optgroup>
-                        `;
+                      <option selected disabled>Select a course to continue</option>
+                      ${courses.map((c) => {
+                        return html`<option value="${c.id}">${c.short_name}: ${c.title}</option>`;
                       })}
                     </select>
-                  </div>
-                  <button class="btn btn-primary" id="saveButton" disabled>Save</button>
-                </form>
+                    <label class="form-label" for="course_instances">
+                      Select a course instance to link:
+                    </label>
+                    <select
+                      id="course_instances"
+                      name="unsafe_course_instance_id"
+                      class="form-select mb-3"
+                    >
+                      <option selected disabled>See above</option>
+                    </select>
+                    <input type="submit" class="btn btn-primary" value="Save" />
+                  </form>
+                </div>
               `}
+
+          <p>
+            <details>
+              <summary>Why don't I see my course or course instances here?</summary>
+              <p>The following PrairieLearn permissions are required to link:</p>
+              <ul>
+                <li>Course: Editor</li>
+                <li>Course instance: Student Data Editor</li>
+              </ul>
```

Comment:
For discussion/consideration:

```suggestion
              <p>You must have course <strong>Editor</strong> and course instance <strong>Student Data Editor</strong> permissions in PrairieLearn to link a course.</p>
```

Before:

<img width="772" alt="Screenshot 2025-01-16 at 15 00 24" src="https://github.com/user-attachments/assets/0315c5ce-fd36-49d8-848a-02b30f8f3dc4" />

After:

<img width="886" alt="Screenshot 2025-01-16 at 15 01 12" src="https://github.com/user-attachments/assets/9ed9eded-0678-4c44-97bd-bd5d2b5a8ad6" />

Reply:
I like the shortened version. Implemented in 6279785a1